src/modules/ogcapi-records/ogcapi-records-implementation/src/main/java/org/geonetwork/ogcapi/configuration/WebConfig.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * (c) 2003 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license,
+ * available at the root application directory.
+ */
+package org.geonetwork.ogcapi.configuration;
+
+import static com.fasterxml.jackson.databind.DeserializationFeature.UNWRAP_SINGLE_VALUE_ARRAYS;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import java.text.SimpleDateFormat;
+import java.util.List;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.http.MediaType;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.web.servlet.config.annotation.ContentNegotiationConfigurer;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+/**
+ * configuration for the Spring Boot app.
+ *
+ * <p>This contains some beans and some overrides for setting up spring.
+ */
+@Configuration
+public class WebConfig implements WebMvcConfigurer {
```

Comment:
Did you consider the option of using the annotation @EnableWebMvc?
 https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/servlet/config/annotation/WebMvcConfigurer.html

Reply:
yes - this was the recommended way to setup the ContentNegotiationConfigurer and provide an objectmapper.

Is there a better way?

Reply:
I am going to merge this - but its actively being worked on.  If you have a better way, I can change it.